src/main/java/chess/dao/ChessDAO.java

```diff
@@ -0,0 +1,20 @@
+package chess.dao;
+
+import chess.entity.ChessGame;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ChessDAO {
```

Comment:
```suggestion
public interface ChessDao {
```

이러한 네이밍은 취향이긴 하지만 저는 위처럼 하는 편인데요.
XmlHttpUrl은 어떻게 나타낼 수 있을까요?

Reply:
음 축약된 단어들이 연속될 경우 대문자만 연속되게 나타나겠네요...!!